gcovr/gcov.py

```diff
@@ -677,7 +683,7 @@ def run_gcov_and_process_files(
     return done
 
 
-def select_gcov_files_from_stdout(out, gcov_filter, gcov_exclude, logger):
+def select_gcov_files_from_stdout(out, gcov_filter, gcov_exclude, logger, chdir, tempdir=None):
```

Comment:
`tempdir=None` never happens, as a `workdir` is always set when executing through the thread. I think these arguments can be made mandatory.

Reply:
Done, will merge (won't rebase yet) when all are completed.